puppet/zulip/manifests/process_fts_updates.pp

```diff
@@ -1,9 +1,9 @@
 class zulip::process_fts_updates {
-  $fts_updates_packages = [
+  exec {'pip_process_fts_updates':
     # Needed to run process_fts_updates
-    'python3-psycopg2', # TODO: use a virtualenv instead
-  ]
-  zulip::safepackage { $fts_updates_packages: ensure => 'installed' }
+    command => 'python3 -m pip install psycopg2',
+    creates => '/usr/local/bin/process_fts_updates',
+  }
```

Comment:
This doesn't create that file?

Reply:
For this, it might be better to just use `scripts/lib/create-production-venv` to create a full Zulip/Django virtualenv which could be reusable for other purposes, rather than doing a custom pip install here.  (I can imagine we'll eventually have other puppet modules that want to be able to just declare they have a Zulip venv).

Reply:
The `create-production-venv` has been run in the previous step before `puppet apply` hasn't it? So this will be just a step to assure that the prod venv has been made?

Reply:
Only if you're running everything on the same system.  We support running `process_fts_updates` on a dedicated database server, and the `create-production-venv` here would be in support of that use case.

Reply:
It would be overkill to have an entire prod-venv on the database server where it only requires `psycopg2`.